tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -110,9 +115,77 @@ const reportsListDetails = async () => {
   return reportDetails;
 };
 
+const collapseSelection = async () => {
+  await (await itemSummary()).click();
+  expect(await (await reportBodyDetails()).isExisting()).to.be.false;
+};
+
+const deleteSelectedReports = async (savedUuids) => {
+  const deleteAllButton = () => $('.action-container .detail-actions .delete-all');
+  const confirmButton = () => $('.btn.submit.btn-danger');
+
+  await (await deleteAllButton()).click();
+  await (await confirmButton()).click();
+  await (await firstReport ()).waitForDisplayed();
+  // make sure the reports are deleted
+  expect(await reportsByUUID(savedUuids[1]).length).to.equal(1);
```

Comment:
How is this testing that reports are deleted? It just tests that this report exists, and makes no assertions about other reports?